config/xinitrc

```diff
@@ -2,6 +2,7 @@ xset -dpms
 xset s off
 xset s noblank
 xrdb ~/.Xresources
+xmodmap ~/.Xmodmap
```

Comment:
Just double-checking - is it safe to assume `xmodmap` is available on locked-down prod images?

From searching, it looks like it's bundled with `x11-xserver-utils`, which I don't think we install directly, but `xmodmap` was available on the QA image I tested with, so I'm guessing it's a transitive dependency.